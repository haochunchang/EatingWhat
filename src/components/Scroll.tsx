import React from "react";

interface Props {
  children?: JSX.Element;
}

const Scroll = (props: Props) => {
  return (
    <div
      style={{ overflowY: "auto", border: "1 px solid black", height: "500px" }}
    >
      {props.children}
    </div>
  );
};

export default Scroll;
