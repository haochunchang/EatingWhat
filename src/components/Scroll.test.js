import React from "react";
import { shallow } from "enzyme";
import Scroll from "./Scroll";

describe("Scroll", () => {
  it("matches Scroll snapshot", () => {
    expect(shallow(<Scroll />)).toMatchSnapshot();
  });
});
