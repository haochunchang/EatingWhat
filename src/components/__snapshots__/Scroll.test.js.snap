// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll matches Scroll snapshot 1`] = `ShallowWrapper {}`;
